aiosmtpd/docs/controller.rst

```diff
@@ -283,7 +283,7 @@ Controller API
    handler, \
    loop=None, \
    *, \
-   ready_timeout=1.0, \
+   ready_timeout, \
```

Comment:
Similar to the changes in `controller.py`; the BaseController does not need a default value.

Later signatures of the `Controller` and `UnixSocketController` classes do indicate the new default value.